src/components/GameListItem.vue

```diff
@@ -16,17 +16,28 @@
         <p>{{ readyText }} players</p>
       </v-col>
       <v-col cols="3" class="list-item__button">
+        <!-- Join Button -->
```

Comment:
Did you test joining/leaving a game to make sure it's updating immediately? If a sync is missed somehow, do we need a poll or regular check in to make sure it's good to go?

Reply:
What do you mean by updating immediately? I sense you're onto a possible bug but I don't fully grasp the issue

Reply:
So imagine opening two tabs, I want to verify that it’s being reactive and that a blast isn’t being missed by either post and leading to stale UX.

Reply:
https://github.com/cuttle-cards/cuttle/issues/373